ports/migrations/0001_initial.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 2.1.7 on 2019-04-11 13:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BuildHistory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('builder_name', models.CharField(db_index=True, max_length=50)),
+                ('build_id', models.IntegerField()),
+                ('status', models.CharField(max_length=50)),
+                ('port_name', models.CharField(db_index=True, max_length=50)),
+                ('time_start', models.CharField(max_length=50)),
+                ('time_elapsed', models.CharField(max_length=50)),
+                ('build_url', models.URLField(default='')),
+                ('watcher_id', models.IntegerField()),
+                ('watcher_url', models.URLField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Category',
+            fields=[
+                ('name', models.TextField(primary_key=True, serialize=False)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Maintainer',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=50)),
+                ('domain', models.CharField(max_length=50)),
+                ('github', models.CharField(db_index=True, max_length=50)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Port',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('portdir', models.CharField(max_length=100)),
+                ('variants', models.TextField(default='')),
```

Comment:
In the future we require a better representation, in particular with respect to statistics collection. The oversimplification may stay for now.